### PR TITLE
Fix MSVC compile error.

### PR DESCRIPTION
**Pull request by shixiongfei**
A good luaffi library, I like it very much.

### CODE REVIEW

File: ffi.c [hunk 1]
```diff
@@ -605,7 +610,7 @@ int cast2ptr(lua_State *L, int idx, void **ptr)
 		switch (type->size) { FLOAT_TYPE_LIST_(COMPLEX_CASE) } \
 	}
 #else
-# define COMPLEX_CASES
+# define COMPLEX_CASES }
```
Comment (z-rui) [inline ffi.c:613]: I think the closing brace on this line (and on line 611) should really be on line 623 to close the switch statement.  This was probably a mistake when I factored the switch cases into macros.

Comment (shixiongfei) [reply ffi.c:613]: if FFI_TARGET_HAS_COMPLEX_TYPE is not defined, the switch statement will not close.
putting the brace on line 623 makes the code look strange :)

Comment (z-rui) [reply ffi.c:613]: I don't think the code will look strange.  Anyway, I have fixed the closing brace.

Comment (shixiongfei) [reply ffi.c:613]: you are right, that's a good place. thank you :D

**Pull request merged**